peripherals/spi/spi_v3.yaml

```diff
@@ -0,0 +1,281 @@
+      TI: [1, "TI SPI protocol"]
+    COMM:
+      FullDuplex: [0, "Full duplex"]
+      Transmitter: [1, "Simplex transmitter only"]
+      Receiver: [2, "Simplex receiver only"]
+      HalfDuplex: [3, "Half duplex"]
+    IOSWP:
+      Disabled: [0, "MISO and MOSI not swapped"]
+      Enabled: [1, "MISO and MOSI swapped"]
+    MIDI: [0, 15]
+    MSSI: [0, 15]
+
+  IER:
+    TSERFIE:
+      Masked: [0, "TSER loaded interrupt masked"]
+      NotMasked: [1, "TSER loaded interrupt not masked"]
+    MODFIE:
+      Masked: [0, "Mode fault interrupt masked"]
+      NotMasked: [1, "Mode fault interrupt not masked"]
+    TIFREIE:
+      Masked: [0, "TI frame format error interrupt masked"]
+      NotMasked: [1, "TI frame format error interrupt not masked"]
+    CRCEIE:
+      Masked: [0, "CRC error interrupt masked"]
+      NotMasked: [1, "CRC error interrupt not masked"]
+    OVRIE:
+      Masked: [0, "Overrun interrupt masked"]
+      NotMasked: [1, "Overrun interrupt not masked"]
+    UDRIE:
+      Masked: [0, "Underrun interrupt masked"]
+      NotMasked: [1, "Underrun interrupt not masked"]
+    TXTFIE:
+      Masked: [0, "Transmission transfer filled interrupt masked"]
+      NotMasked: [1, "Transmission transfer filled interrupt not masked"]
+    EOTIE:
+      Masked: [0, "End-of-transfer interrupt masked"]
+      NotMasked: [1, "End-of-transfer interrupt not masked"]
+    DPXPIE:
+      Masked: [0, "Duplex transfer complete interrupt masked"]
+      NotMasked: [1, "Duplex transfer complete interrupt not masked"]
+    TXPIE:
+      Masked: [0, "TX space available interrupt masked"]
+      NotMasked: [1, "TX space available interrupt not masked"]
+    RXPIE:
+      Masked: [0, "RX data available interrupt masked"]
+      NotMasked: [1, "RX data available interrupt not masked"]
+
+  SR:
+    CTSIZE: [0, 65535]
+    RXWNE:
+      LessThan32: [0, "Less than 32-bit data frame received"]
+      AtLeast32: [1, "At least 32-bit data frame received"]
+    RXPLVL:
+      ZeroFrames: [0, "Zero frames beyond packing ratio available"]
+      OneFrame: [1, "One frame beyond packing ratio available"]
+      TwoFrames: [2, "Two frame beyond packing ratio available"]
+      ThreeFrames: [3, "Three frame beyond packing ratio available"]
+    TXC:
+      Ongoing: [0, "Transmission ongoing"]
+      Completed: [1, "Transmission completed"]
+    SUSP:
+      NotSuspended: [0, "Master not suspended"]
+      Suspended: [1, "Master suspended"]
+    TSERF:
+      NotLoaded: [0, "Additional number of SPI data to be transacted not yet loaded"]
+      Loaded: [1, "Additional number of SPI data to be transacted was reloaded"]
+    MODF:
+      NoFault: [0, "No mode fault detected"]
+      Fault: [1, "Mode fault detected"]
+    TIFRE:
+      NoError: [0, "TI frame format error detected"]
+      Error: [1, "TI frame format error detected"]
+    CRCE:
+      Match: [0, "No CRC error detected"]
+      NoMatch: [1, "CRC error detected"]
```

Comment:
Do you think `NoError`/`Error` would more clearly match the field name and RM?

Reply:
ACK.